app/serializers/curator/filestreams/file_set_serializer.rb

```diff
@@ -3,7 +3,20 @@
 module Curator
   class Filestreams::FileSetSerializer < CuratorSerializer
     schema_as_json do
-      attributes :ark_id
+      attributes :file_name_base, :position
+
+      node :file_set_of, target: :key do
+        attributes :ark_id
+      end
+
+      node :pagination do
+        attributes :page_label, :page_type, :hand_side
+      end
+
+      node :metastreams, target: :key do
+        has_one :administrative, serializer: 'Curator::Metastreams::AdministrativeSerializer'
+        has_one :workflow, serializer: 'Curator::Metastreams::WorkflowSerializer'
+      end
     end
   end
 end
```

Comment:
`FileSetSerializer` should include an attribute `file_set_type` that indicates its subclass (preferably without the namsepace, using `class.name.demodulize`).

There's no corresponding spec file for this class. Or is this covered in `shared/inherited_serializers.rb`?